src/harvest_translator.py

```diff
@@ -38,7 +38,22 @@
         'units'
     ]
 )
+
```

Comment:
I don't think this is a good structure to add the expected output to the same string because then it isn't clear which harvester it matches up to and will become increasing confusing. This is also more consistent with most coding guidelines that comments belong with the code which they directly relate to. In this case, this expected output comment should be moved to be directly above the precip_translator.

Reply:
Thanks, Jessica. I updated the PR using a new style throughout.